consensus/config/src/parameters.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::time::Duration;
+
+use serde::{Deserialize, Serialize};
+
+/// Operational configurations of a consensus authority.
+///
+/// All fields should tolerate inconsistencies among authorities, without affecting safety of the
+/// protocol. Otherwise, they need to be part of Sui protocol config or epoch state on-chain.
+///
+/// NOTE: default values should make sense, so most operators should not need to specify any field.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct Parameters {
+    /// Time to wait for parent round leader before sealing a block.
+    #[serde(default = "Parameters::default_leader_timeout")]
+    pub leader_timeout: Duration,
+}
+
+impl Parameters {
+    pub fn default_leader_timeout() -> Duration {
+        Duration::from_millis(200)
```

Comment:
Not that it matters a lot right now, but the lowest I've used was 250ms on our experiments, so we could probably fallback on that for the time being

Reply:
Yeah this is meant to be a placeholder value. But setting it to 250ms for now.